URLDemonstrationAPP/app.R

```diff
@@ -129,7 +139,30 @@ server <- function(input, output) {
                                          face="bold",
                                          angle=0))
   }, res = 96, bg="transparent")
-  
+  output$networkgraph <- renderPlot({
+    data() %>% 
+      unnest_tokens(bigram, fulltext_org, token = "ngrams", n = 2) %>% 
+      separate(bigram, c("word1", "word2"), sep = " ") %>% 
+      filter(!word1 %in% stopord$word) %>%
+      filter(!word2 %in% stopord$word) %>%
+      filter(!word1 %in% stop_words$word) %>%
+      filter(!word2 %in% stop_words$word) %>%
+      count(word1, word2, sort = TRUE) -> bigrams_count
+    
+    bigram_graph <- bigrams_count %>%
+      filter(n > slider_input()) %>%
+      graph_from_data_frame()
+    
+    a <- grid::arrow(type = "closed", length = unit(.15, "inches"))
+    
+    ggraph(bigram_graph, layout = "fr") +
+      geom_edge_link(aes(edge_alpha = n), show.legend = FALSE,
+                     arrow = a, end_cap = circle(.07, 'inches')) +
+      geom_node_point(color = "#FFFFF0", size = 3) +
```

Comment:
I think some code here is deprecated. I got this warning when running the app: 

Warning: Using the `size` aesthetic in this geom was deprecated in ggplot2 3.4.0.
ℹ Please use `linewidth` in the `default_aes` field and elsewhere instead.

Could you please change this?